game/scripts/vscripts/heroes/antimage/antimage_mana_void.lua

```diff
@@ -0,0 +1,121 @@
+-- Author: Shush
+-- Date: 29/11/2017
+
+-------------------------------
+--   Anti Mage's Mana Void   --
+-------------------------------
+lua_antimage_mana_void = class({})
+LinkLuaModifier("modifier_lua_mana_void_stunned", "antimage/antimage_mana_void", LUA_MODIFIER_MOTION_NONE)
+
+function lua_antimage_mana_void:GetAbilityTextureName()
+    return "antimage_mana_void"
+end
+
+function lua_antimage_mana_void:OnAbilityPhaseStart()
+    if IsServer() then
+        self:GetCaster():EmitSound("Hero_Antimage.ManaVoidCast")
+        return true
```

Comment:
Should this return not be outside of the server block?

Reply:
The IsServer() isn't supposed to be there at all as OnAbilityPhaseStart is only called server-side anyway.